### ISSUE TITLE
UI Automation of Sliders no longer updates bound values on Android

### ISSUE THREAD

**Issue by DamianMehers** [xamarin/Xamarin.Forms#2800]
### Description

[This change](https://github.com/xamarin/Xamarin.Forms/pull/1756) seems to have broken UI Automation of sliders, specifically the update of bound values

### Steps to Reproduce

1.  Create a new Xamarin Forms Android app with a slider whose value is databound, and that has an automation ID
2. Add a UITest project and [set the slider value](https://developer.xamarin.com/api/member/Xamarin.UITest.IApp.SetSliderValue/p/System.String/System.Double/)

### Expected Behavior
The bound value is updated

### Actual Behavior
The bound value is not updated, although the slider moves

### Basic Information

- Version with issue: Xamarin Forms 3
- Last known good version: Xamarin Forms 2
- Platform Target Frameworks: Android
 ### Reproduction Link
[SliderUITest.zip](https://github.com/xamarin/Xamarin.Forms/files/2027014/SliderUITest.zip)

**Comment by PureWeen** [comment]
@DamianMehers  when you say Forms 2 do you have a particular version you are referring to?  I tested your sample back to 2.4 and still saw the same behavior. I doesn't seem to be a regression.

**Comment by DamianMehers** [comment]
Sorry, I oversimplified the example.  I'm attaching an updated example, which fails in XF 3, and works in XF2.
[SliderUITest2.zip](https://github.com/xamarin/Xamarin.Forms/files/2030358/SliderUITest2.zip)

My workaround is to copy the [SliderRenderer](https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Platform.Android/Renderers/SliderRenderer.cs) and [FormsSeekBar](https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Platform.Android/Renderers/FormsSeekBar.cs) from XF3 and comment the line that checks for tracking (which I guess only happens if a person is manually pressing the SeekBar):

``` C#
void SeekBar.IOnSeekBarChangeListener.OnProgressChanged(SeekBar seekBar, int progress, bool fromUser)
		{
//			if (_isTrackingChange)
				((IElementController)Element).SetValueFromRenderer(Slider.ValueProperty, Value);
		}
```

Note that even in XF 2.0 I had a workaround which moved the slider twice, the first time being ignored (see the example).

**Comment by samhouts** [comment]
closed by #10204
